packages/polkadart_scale_codec/test/test_types/unsigned_int_test.dart

```diff
@@ -66,7 +69,7 @@ void main() {
         );
       });
 
-      test('Should throw InvalidSizeException when value is greater than 256',
+      test('Should throw InvalidSizeException when value is greater than 255',
```

Comment:
magic numbers again. 
```suggestion
      test('Given an 8 bit decoder when value can't be represented it should throw',
```